### PR TITLE
Clarify usage of Ingress backend.servicePort

### LINKED ISSUE #2244: [GLBC] purpose of backend.servicePort
An Ingress in GKE creates an HTTPS GCE Load Balancer, the Backend Service for which targets the Service's NodePorts. So what is the purpose of `servicePort`?

The only instances in the code are examples, except for the note in `examples/health_checks/README.md`. If it's only used with readiness probes, that should be mentioned here https://github.com/kubernetes/contrib/blame/master/ingress/controllers/gce/README.md#L54.

Otherwise, it seems that the Service name should be all that's needed to locate the NodePort.
```
$ grep -r servicePort ingress/controllers/gce/
ingress/controllers/gce//BETA_LIMITATIONS.md:    servicePort: 80
ingress/controllers/gce//BETA_LIMITATIONS.md:    servicePort: 80
ingress/controllers/gce//examples/health_checks/health_check_app.yaml:          servicePort: 80
ingress/controllers/gce//examples/health_checks/health_check_app.yaml:          servicePort: 80
ingress/controllers/gce//examples/health_checks/health_check_app.yaml:          servicePort: 80
ingress/controllers/gce//examples/health_checks/README.md:* The readiness probe must be exposed on the port matching the `servicePort` specified in the Ingress
ingress/controllers/gce//examples/https/tls-app.yaml:    servicePort: 80
ingress/controllers/gce//ingress-app.yaml:    servicePort: 80
ingress/controllers/gce//ingress-app.yaml:          servicePort: 80
ingress/controllers/gce//ingress-app.yaml:          servicePort: 80
ingress/controllers/gce//ingress-app.yaml:          servicePort: 80
ingress/controllers/gce//README.md:12.          servicePort: 80
ingress/controllers/gce//README.md:          servicePort: 80
ingress/controllers/gce//README.md:          servicePort: 80
ingress/controllers/gce//README.md:    servicePort: 80
ingress/controllers/gce//README.md:    servicePort: 80
ingress/controllers/gce//README.md:    servicePort: 80
```

### PR DESCRIPTION
**Pull request by jrynyt**
Resolves https://github.com/kubernetes/contrib/issues/2244

### CODE REVIEW

**Comment by jrynyt** [comment]
I signed it!

**Comment by aledbf** [comment]
@jrynyt the development of ingress is now located in https://github.com/kubernetes/ingress
Please check the GCE code in that repo.

File: ingress/controllers/gce/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ __Lines 5-7__: Ingress Spec has all the information needed to configure a GCE L7
 
 __Lines 8-9__: Each http rule contains the following information: A host (eg: foo.bar.com, defaults to `*` in this example), a list of paths (eg: `/hostless`) each of which has an associated backend (`test:80`). Both the `host` and `path` must match the content of an incoming request before the L7 directs traffic to the `backend`.
 
-__Lines 10-12__: A `backend` is a service:port combination. It selects a group of pods capable of servicing traffic sent to the path specified in the parent rule.
+__Lines 10-12__: A `backend` is a service:port combination. It selects a group of pods capable of servicing traffic sent to the path specified in the parent rule. The `port` is the desired `spec.ports[*].port` from the Service Spec -- Note, though, that the L7 actually directs traffic to the corresponding `NodePort`.
```
Comment (bprashanth) [inline ingress/controllers/gce/README.md]: this is only the case on GCE, please includ "Note though, that on some cloudproviders like GCE the cloud lb actually directs ..."

Comment (jrynyt) [inline ingress/controllers/gce/README.md]: Isn't this GCE-specific documentation? The above lines talk about "the L7," so I just followed that.

I'll resubmit this in the ingress repo.

**Comment by jrynyt** [comment]
moved this over to https://github.com/kubernetes/ingress/pull/173, closing.

**Pull request closed without merging**